great_expectations_cloud/agent/agent.py

```diff
@@ -550,9 +551,3 @@ def _update_headers_agent_patch(
         # TODO: this is relying on a private implementation detail
         # use a public API once it is available
         http._update_headers = _update_headers_agent_patch
-
-
-class GXAgentError(Exception): ...
-
-
-class GXAgentConfigError(GXAgentError): ...
```

Comment:
Moving to `exceptions` class to prevent circular-import when using error in `EventHandler`.